extensions/data-transfer/portability-data-transfer-google/src/main/java/org/dataportabilityproject/datatransfer/google/calendar/GoogleCalendarExporter.java

```diff
@@ -106,7 +106,9 @@ private static CalendarEventModel convertToCalendarEventModel(String id, Event e
   @Override
   public ExportResult<CalendarContainerResource> export(
       UUID jobId, TokensAndUrlAuthData authData, Optional<ExportInformation> exportInformation) {
-    if (exportInformation.isPresent()) {
+    if (!exportInformation.isPresent()) {
+      return exportCalendars(authData, Optional.empty());
+    } else {
```

Comment:
nit: omit the else since we return from the if.

Reply:
Whoops, I'll get it in the next one.